public/countries.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<link rel="icon" href="assets/image/favicon.ico">
+
+		<title>OpenFisca - Available country packages</title>
+
+		<link rel="stylesheet" type="text/css" href="assets/css/main.css">
+	</head>
+
+	<body>
+		<header>
+			<a href="./index.html">
+				<img id="logo" src="assets/image/logo-openfisca.svg" alt="OpenFisca">
+			</a>
+
+			<h1>Available country packages</h1>
+
+			<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
```

Comment:
I don't think this CTA is relevant here

Reply:
Removed. 
Shouldn't we remove the CTA at the end of the page as well?